### ISSUE TITLE
Steel & Ironsight not working correctly

### ISSUE THREAD

**Issue by whiddenmason** [leafreynolds/cosmere#125]
### Issue description

When at certain angles or even when right next to a metal source, ironsight and steelsight don't show anything there, meaning I can't push or pull on it.

### Steps to reproduce

1. obtain any level of iron or steel allomancy and burn them. 2. Find a sizable clump of metal source blocks, (Ores, blocks, etc) 3. get close and walk around and on the metal sources. It should be found that they often won't be visible with steelsight or ironsight.

### Minecraft version

1.20.1 (Latest)

### Forge version

47.3.0

### Cosmere version

Cosmere-1.20.1-47.3.0-0.7.95.jar

**Comment by Gerbagel** [comment]
Are these blocks beneath the ground? If they are, it's probably intentional. If not, then that's a pretty big problem. I would check myself, but I'm at work at the moment

**Comment by R0b0r0b** [comment]
I will check 🫡

**Comment by R0b0r0b** [comment]
I have checked
Seems like this problem only happens if the blocks are sufficiently deep underground, so yeah, likely to be intentional

**Comment by Gerbagel** [comment]
Yep, that's intentional. In the books, steelsight is significantly hampered by physical objects like walls. We've mimicked that (rather unintuitively) by giving blocks "resistance". I'll leave this open in case this is still a problem, but I'll close it later once I've double-checked for issues

**Comment by whiddenmason** [comment]
No, this issue has been happening to me for blocks above ground. I have literally stood right on top of one and it didn't show up.

**Comment by R0b0r0b** [comment]
Huh, that's very weird - hasn't been happening to me

**Comment by R0b0r0b** [comment]
Do you have optifine? Optifine has known compatibility issues with pretty much all mods which make changes to how Vanilla runs

**Comment by whiddenmason** [comment]
Nope, no optifine. I am running a modpack though. I'll be able to provide a mod list later in the day if needed.

**Comment by Gerbagel** [comment]
Okay, have looked at it in the dev environment, there seems to be no issues other than maybe slightly shoddy selection code.

However, I think I remember something like this from before in 1.19.2. Unsure what caused it, but if you're using any kind of graphical mods such as Embeddium and Oculus, things may get wacky. We unfortunately can't guarantee full compatibility with any mod that changes Minecraft's rendering.

**Comment by whiddenmason** [comment]
Okay, I'll check what mods I have after school, I don't think I have any mods that would change rendering, but I will check for you and send the modlist.

**Comment by whiddenmason** [comment]
[COMSEER PORT.zip](https://github.com/user-attachments/files/17909973/COMSEER.PORT.zip)
(This is the modpack for curseforge, title is weird cus my friend made it.)

**Comment by Gerbagel** [comment]
Hey @whiddenmason sorry for taking so long, but I'm seeing no issues with that modpack. I made a huge cluster, tried on individual blocks, and nothing. I can't figure it out. I think I'll put this bug report to the side for now

**Comment by whiddenmason** [comment]
Okay, that's alright. I'm not quite sure what the issue is. I also started a new world and haven't used allo steel or iron in a while so maybe something changed. Thank you for your time.
